openadapt/app/main.py

```diff
@@ -0,0 +1,78 @@
+import threading
+
+from nicegui import app, ui
+
+from openadapt import replay, visualize
+from openadapt.app.cards import recording_prompt, select_import, settings
+from openadapt.app.objects.console import Console
+from openadapt.app.util import clear_db, on_export, on_import
+
+
+def run_app():
+    SERVER = "127.0.0.1:8000/upload"
```

Comment:
Can you please move this to a global constant in this file (i.e. move it up a few lines)?